app/service.py

```diff
@@ -183,4 +183,34 @@ def verify_answer(answerID):
 
     result = 'spravna' if answer.correct_answers else 'spatna'
 
-    return jsonify({"Vysledek": "Vase odpoved byla {}".format(result)})
```

Comment:
nevim, jestli to znas, ale od Python 3.6 (myslim) se da pouzivat f string notace... 
takhle by to vypadalo: 
f"Vase odpoved byla {result}."
Prijde mi to prehlednejsi a rychlejsi, ale neni to uprava, jen takovy mensi comment :)